boltdesignsystem/bolt
# ci: setup 2 different unit tests jobs if on master

## Jira

BDS-1607

## Summary

Enables `yarn test:js:quick` (just tests for packages that have changed) to run in CI when not on `master` and sets up `yarn test:js` to run on `master`.

## Details

See #1226 

## How to test

The branch `feature/example-change` has this code and also a simple change in the button; see [it's Travis build](https://travis-ci.com/bolt-design-system/bolt/builds/122663328) and note that only the button test runs.
